src/utils/sendEmail.ts

```diff
@@ -0,0 +1,17 @@
+import * as nodemailer from 'nodemailer';
+export const sendEmail = async (email: string, link: string) => {
+    const transporter = nodemailer.createTransport({
+        service:'gmail',
+        auth: {
+            user: process.env.email,
+            pass: process.env.emailPass,
+        },
+    });
+     await transporter.sendMail({
+        from: process.env.email,
+        to: email,
+        subject: 'Hello ',
+        text:'hiiiii',
+       html: `<b>Hello world?</b> <a href="${link}">confirm Email</a>`,
+    });
+};
```

Comment:
Convert it into service to initiate transport only once, and html should be a file, text should be `hiii` all constants should be come from one place